docs/iot-gateway/config/opc-ua.md

```diff
@@ -220,6 +392,51 @@ This part of configuration will look like:
         ]
 ```
 
+**Let’s look an example.**
+
+Suppose you want to set the value of the **"deviceName"** attribute. Currently, the attribute hasn't any value.
+
+In the OPC-UA Connector configuration file (opcua.json) change **"attributeOnDevice"** value to the full path to the node "deviceName".
+
+In this example it is **"Root\\.Objects\\.Simulation\\.deviceName"**.
+
+{:refdef: style="text-align: left;"}
+![image](/images/gateway/gateway-opc-ua-attributes-updates-2.png)
+{: refdef}
+
+Our **attributes_updates** section would look like this:
+
+```json
+  "attributes_updates": [
+    {
+      "attributeOnThingsBoard": "deviceName",
+      "attributeOnDevice": "Root\\.Objects\\.Simulation\\.deviceName"
+    }
+  ]
+```
+
+{:refdef: style="text-align: left;"}
+![image](/images/gateway/gateway-opc-ua-attributes-updates-1.png)
+{: refdef}
+
+In the ThingsBoard instance, go to **"Shared attributes"** and create a new attribute for your device.
```

Comment:
Maybe it will be better:
"Go to **"Shared attributes"** and create a new one for your device in the ThingsBoard instance"

Reply:
replaced